src/parserror.js

```diff
@@ -311,9 +324,10 @@ class Parserror {
    * @return {ParserrorWrapper}
    */
   wrap(cases = [], scopes = []) {
-    return (error) => this.parse(error, ({
+    return (error, fallback = null) => this.parse(error, ({
```

Comment:
The pattern for the `parse()` method is to have `cases`, `scopes` and `fallback` in the `options` parameter. Given that the `wrap()` method sets its own parameters to be the values stored within the `options` parameter in the `parse()` method, why wouldn't fallback be a third parameter for `wrap()`? Wouldn't it be more consistent to have something like this;
```
wrap(cases = [], scopes = [], fallback = null) {
  return (error) => this.parse(error, ({
    cases,
    scopes,
    fallback,
  }));
}
```
(?)

Reply:
We already discussed this offline but I'll add it here too as reference:

1. There's no "pattern" that is being ignored; `parse` works like that because all three things (`cases`, `scopes` and `fallback`) are optional and unrelated.
2. `fallback` was added with the idea of being used on a single error parsing; if you are using `wrap` or `wrapForScopes`, you intend to parse multiple errors on a module/context, and it doesn't sound like a good idea to have multiple errors doing fallback to the same message.
3. Adding support for `fallback` on `wrap` and `wrapForScopes` is not hard, so I'll do it, but I believe using it on the wrapped function is a better approach.